src/lib/components/table/EditableCell.tsx

```diff
@@ -90,6 +90,8 @@ export const EditableCell = ({
     isShowInputValue &&
     Number(textRef.current?.scrollWidth) > Number(textRef.current?.clientWidth);
 
+  useEffect(() => setInputValue(initialValue), [initialValue]);
```

Comment:
@songwongtp Last time I remembered, we just remove this for fixing some bug, right?

Reply:
probably a different component